### PR TITLE
Pull request created to insert changes as previous one was closed after receiving approval

### PR DESCRIPTION
**Pull request by DarioAlessioR**
After receiving approval of the last pull request review, branch portfolio1 was merged into the master branch, and the pull request was closed.

After that, the reviewer sent a new email and notified of a linter issue.

The issue was fixed and changes were committed, so this pull request intends to serve as the way to ask again for project approval.

In addition to fining linter issue, image files were updated.

### CODE REVIEW

**Comment by DarioAlessioR** [comment]
The project was updated with the section "About myself" added.  This section is no part of Figma's template # 4, which is the selected model for this project.

However, Microvers instructions for this project establish the need for an "About myself" section, and so it was added to fulfill this instruction.

Linters were again applied and resulted ok.

Project ready to ask for approval of reviewer.

Thank you in advance.

File: PortafolioMobile.html [hunk 1]
```diff
@@ -4,12 +4,13 @@
...
       <title>Dario Alessio Portfolio Page</title>  
-      <link
+      <script src="https://kit.fontawesome.com/8bfb3ed134.js" crossorigin="anonymous"></script>
+      <!--<link
         rel="stylesheet"
         href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
         integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay"
         crossorigin="anonymous"
-      />          
+      />-->
```
Comment (CollinsTatang) [inline PortafolioMobile.html]: - [ ] Please remove all commented code from your codebase to make it clean and professional.

File: PortafolioMobile.html [hunk 2]
```diff
@@ -130,15 +131,15 @@ <h3>"Teach yourself social networks" website</h3>
...
+            <img src="images/portfolio/SocialMediaLogos.jpg" alt="image SocialMediaLogos.jpg" />
           </div>
 
         </article>
       
         <article class="art_img_left">
 
           <div class="work-image">
-            <img src="images/portfolio/Website.png" alt="image Website.png" />
+            <img src="images/portfolio/TeachSeniors.png" alt="image TeachSeniors.png" />
```
Comment (CollinsTatang) [inline PortafolioMobile.html:137]: Hello @DarioAlessioR 
Geat job implementing the different features of this milestone in the work section the following are required:

- [ ] Please remove the different image `img tag` and implement it with a proper `background-image style`

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @DarioAlessioR, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required Changes ♻️

- [ ] Please kindly rename your `PortafolioMobile.html` file to `index.html` to make your project more professional.

- [ ] I will like to mention this here since the README file was not touched. Please kindly follow the [MICROVERSE  README TEMPLATE](https://github.com/microverseinc/readme-template) to add the different headers in your readme file to make it look professional.

- [ ] Your design looks good but you can do better. Please follow the [figmal design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) and make it responsive to different screen size. 
**Your design is not responsive to mobile view**
badview
### **Take a look at figmal design**
goodview


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DarioAlessioR** [comment]
Hi @CollinsTatang . Thank you for your review and your kind advice on how to improve my project.

I would like to ask you for more detail about the README file comment you wrote.

Certainly, the README file was not touched from the previous pull request to this one.

However, I think it contains all the sections mentioned in the Microverse template. Those sections of the template of which I have no information or that don't apply to this project were left in the file with no answer or with just a comment that it doesn't apply.

I'll greatly appreciate it if you please can help me with more information regarding this point, just to be able to make changes accordingly and make no new mistakes about it.

Thank you in advance for your kind response.

**Comment by DarioAlessioR** [comment]
> Hi @CollinsTatang . Thank you for your review and your kind advice on how to improve my project.
> 
> I would like to ask you for more detail about the README file comment you wrote.
> 
> Certainly, the README file was not touched from the previous pull request to this one.
> 
> However, I think it contains all the sections mentioned in the Microverse template. Those sections of the template of which I have no information or that don't apply to this project were left in the file with no answer or with just a comment that it doesn't apply.
> 
> I'll greatly appreciate it if you please can help me with more information regarding this point, just to be able to make changes accordingly and make no new mistakes about it.
> 
> Thank you in advance for your kind response.

Hi @CollinsTatang . After reviewing again I realize that my README file was without the proper headers template format, although it has the sections needed. So I'm changing it to give the exact format as the template.

If there's something else to be improved apart from the headers format, please don't hesitate to point to it in your appreciated response to this comment.

Thank you in advance.

**Comment by DarioAlessioR** [comment]
Project updated according to reviewer requirements  in order to:

1) Change the HTML file name to index.html
2) Apply Microverse README template format to the README file.
3) Make the design responsive to different screen sizes.
4) Erase all comments from code in HTML and CSS files.
5) Run linters again with result ok.

Project ready to ask for approval in 2nd revision.

Thank you in advance.

**Review by JuliCarracedo — APPROVED** [review]
# Status: Approved ✔️
Hi @DarioAlessioR , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DarioAlessioR** [comment]
Proceed to merge branch portfolio1 into the master branch after approval of pull request by reviewer @JuliCarracedo in the second revision.

Thank you.

**Pull request merged**